sdk/rust/clients/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "dstack-sdk"
+name = "dstack-sdk-clients"
```

Comment:
Thank you for the contribution. I'd like to keep the crate name `dstack-sdk-clients` as `dstack-sdk`, for compatibility with other users and consistency with other language implementations.